doc/Development_Documentation/21_Deployment/03_Multi_Environment.md

```diff
@@ -117,14 +117,20 @@ When running `./bin/console` application, set the environment by `--env=dev`.
 
 ### `.env`
 
-Pimcore loads a `.env` file if it exists. See [DotEnv Component Documentation](https://symfony.com/doc/3.4/components/dotenv.html)
-for details.
+Pimcore loads environment variables from `.env` files if they exist.
+It's good practice to add defaults to `.env`, which is committed to your repository, and local overrides to `.env.local`, which is not committed.
+See [Symfony Configuration Based on Environment Variables](https://symfony.com/doc/current/configuration.html#configuration-based-on-environment-variables) for details.
 
 ```
 # .env
 PIMCORE_ENVIRONMENT=dev
 ```
 
+```
+# .env.local
+PIMCORE_ENVIRONMENT=prod
```

Comment:
I would keep the value "prod" in .env(default value) and "dev" in env.local (local overrides).

Reply:
ok nevermind! Symfony also suggests to keep prod values in env.local